render/id.go

```diff
@@ -39,12 +39,21 @@ func NewDerivedPseudoNode(id string, node report.Node) report.Node {
 
 // NewDerivedExternalNode figures out if a node should be considered external and creates the corresponding pseudo node
 func NewDerivedExternalNode(n report.Node, addr string, local report.Networks) (report.Node, bool) {
+	id, ok := externalNodeID(n, addr, local)
+	if !ok {
+		return report.Node{}, false
+	}
+	return NewDerivedPseudoNode(id, n), true
+}
```

Comment:
Inverting the condition would allow you to inline the first line.

Reply:
function NewDerivedExternalNode has been removed now